frontend/public/components/dashboard/dashboards-page/overview-dashboard/queries.ts

```diff
@@ -23,22 +23,23 @@ export enum OverviewQuery {
 
 const top25Queries = {
   [OverviewQuery.PODS_BY_CPU]:
-    'topk(25, sort_desc(sum(rate(container_cpu_usage_seconds_total{container="",pod!=""}[5m])) BY (pod, namespace)))',
+    'topk(25, sort_desc(sum(avg_over_time(pod:container_cpu_usage:sum{container="",pod!=""}[5m])) BY (pod, namespace)))',
   [OverviewQuery.PODS_BY_MEMORY]:
-    'topk(25, sort_desc(sum(container_memory_working_set_bytes{container="",pod!=""}) BY (pod, namespace)))',
+    'topk(25, sort_desc(sum(avg_over_time(container_memory_working_set_bytes{container="",pod!=""}[5m])) BY (pod, namespace)))',
   [OverviewQuery.PODS_BY_STORAGE]:
-    'topk(25, sort_desc(avg by (pod, namespace)(irate(container_fs_io_time_seconds_total{container="POD", pod!=""}[1m]))))',
-  [OverviewQuery.NODES_BY_CPU]: 'topk(5,sort_desc(instance:node_cpu_utilisation:rate1m))',
+    'topk(25, sort_desc(sum(avg_over_time(pod:container_fs_usage_bytes:sum{container="", pod!=""}[5m])) BY (pod, namespace)))',
+  [OverviewQuery.NODES_BY_CPU]:
+    'topk(25, sort_desc(avg_over_time(instance:node_cpu:rate:sum[5m])))',
   [OverviewQuery.NODES_BY_MEMORY]:
-    'topk(25,sort_desc(node_memory_MemTotal_bytes{job="node-exporter"} - node_memory_MemAvailable_bytes{job="node-exporter"}))',
+    'topk(25, sort_desc(avg_over_time(node_memory_Active_bytes[5m])))',
```

Comment:
I am not 100% sure this is correct, @paulfantom may you have a look?

Reply:
FYI the same query is used in Node details page

Reply:
`node_memory_Active_bytes` will give only RSS memory and this isn't something we use anywhere. We either use WSS or (MemTotal-MemAvailable) and for that reason I don't think we should change it here.

Reply:
reverted back to MemTotal-MemAvailable and using the same query in node details too